src/xunit.performance.api/ISpan.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    /// <summary>
+    /// Defines an interval with a mechanism for testing if a value falls in the range.
+    /// </summary>
+    /// <typeparam name="T">The type span space.</typeparam>
+    internal interface ISpan<T>
```

Comment:
You seem to refer to this as an "interval" in comments, so why not call it `IInterval<T>`?

Reply:
I use span and interval in different places.

---
In reply to: [145194293](https://github.com/Microsoft/xunit-performance/pull/242#discussion_r145194293) [](ancestors = 145194293)